JMeterAKSPipeline/DeleteLoadTestInfra.ps1

```diff
@@ -0,0 +1,41 @@
+param(
```

Comment:
please use full name for file names i.e. DeleteLoadTestInfrastructure.ps1 and DeleteLoadTestInfrastructure.ps1

Reply:
done